internal/proxy/proxy_config.go

```diff
@@ -75,6 +76,7 @@ type OptionsConfig struct {
 	HeaderOverrides map[string]string `yaml:"header_overrides"`
 	SkipAuthRegex   []string          `yaml:"skip_auth_regex"`
 	AllowedGroups   []string          `yaml:"allowed_groups"`
+	TLSVerify       bool              `yaml:"tls_verify"`
```

Comment:
(nit) Can we add a unit test for this config value to ensure that it is being set?

Reply:
Absolutely. I was having trouble understanding your test suite when I first wrote the PR, but a couple PRs in, I think it should be no problem.